src/components/Table/Table.test.tsx

```diff
@@ -1,36 +1,32 @@
-import { shallow } from "enzyme";
+import { render, screen } from "@testing-library/react";
 import React from "react";
 
 import Table from "./Table";
 
 describe("Table", () => {
   it("renders", () => {
-    const wrapper = shallow(
+    const { container } = render(
       <Table>
         <thead></thead>
       </Table>
     );
-    expect(wrapper).toMatchSnapshot();
+    // eslint-disable-next-line testing-library/no-node-access, testing-library/no-container
+    expect(container.firstChild).toMatchSnapshot();
```

Comment:
This doesn't feel like a good solution to me - based on the comments from Bartek, obviously getting it by role "grid" isn't ideal either so the best thing to do here might just be to use a data id.

Reply:
Either that or match the container directly to the snapshot I guess

Reply:
I've updated these tests to use a test id.